packages/console/src/pages/UserDetails/UserSettings/index.tsx

```diff
@@ -48,35 +50,42 @@ function UserSettings() {
     if (isSubmitting) {
       return;
     }
-
-    const { customData: inputCustomData, name, avatar } = formData;
+    const { identities, id: userId } = user;
+    const { customData: inputCustomData, username, primaryEmail, primaryPhone } = formData;
+
+    if (!username && !primaryEmail && !primaryPhone && Object.keys(identities).length === 0) {
```

Comment:
nit: can we extract this predicate into a util function?

Reply:
We can, but is there anywhere else also needs this?